CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ the License.  You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 
-Unless required by applicable law or agreed to in writing, software
+Unless required by applicable law or agreed to in writing, softwares
```

Comment:
Please revert this change.

1) The plural form of software is software - so the original is correct.  
2) This is boilerplate from the ASF, if it would need to be changed it should be done at the ASF level.

Reply:
Oops! Sorry @EdColeman, sorry my bad. I didn't do that intentionally as I was just going through the readme in vs code and it might have pressed some key. I am sorry for this change. I'll revert is ASAP.